.travis.yml

```diff
@@ -1,27 +1,57 @@
-sudo: false
 language: python
 
 env:
-    - CONDA="python=2.7"
-    - CONDA="python=3.3"
-    - CONDA="python=3.4"
+  - CONDA_PYTHON_VERSION="2"
+  - CONDA_PYTHON_VERSION="3"
```

Comment:
> How so? We are testing from python 2.7 to Python 3.4 (3.5 now) skiping, on purpose 3.2 and 3.1. I'd rather keep the current configuration then just declaring major versions.

I haven't yet found / added support for ``rtree`` Python 3.x. Tests were passing when I very much expected a failure similar to the Python 3 failures now: https://travis-ci.org/NESII/pyugrid/jobs/87780816#L409. Does that make sense? 

> However, if that is implemented you could try splitting Travis into a build and test component. Here is an example: https://github.com/SciTools/iris/blob/master/.travis.yml#L14-L19

That's a nice example! Are they testing Python 3? It isn't listed in the Python versions. It looks like they are also installing miniconda in a similar way to the in-question .travis.yml: https://github.com/SciTools/iris/blob/master/.travis.yml#L34.

> In this case, the recipe is pretty trivial, so not a big deal either way -- but would the recipe reference a source download in the usual way, or build from the source it is embedded in...

I addressed this by using environment variables in the build script (updated recently to use "get"): https://github.com/NESII/pyugrid/blob/259ad633d22180aeb050f71186562da16f9cc818/conda-recipe/meta.yaml#L3. The source/branch/tag are relative to the Git directory. The build will succeed regardless of branch or origin: https://travis-ci.org/NESII/pyugrid/jobs/88165831.